frontend/src/app/libs/common-components/components/tasks-creator/tasks-creator/tasks-creator.component.html

```diff
@@ -0,0 +1,160 @@
+<form [formGroup]="taskCreatorControl" class="tasks-creator">
+  <ng-container *ngIf="!isListEmpty; else emptyList">
+    <div class="tasks-creator__header event-gradient">
+
+      <div *ngIf="!isTaskNameEditMode" class="tasks-creator__name-editor">
+        <span class="tasks-creator__task-name">{{ taskName }}</span>
+        <app-icon-button
+          class="tasks-creator__header-button"
+          (click)="enableTaskNameEditMode()"
+          startIcon="mode"
+          iconColor="white"
+          iconSize="small">
+        </app-icon-button>
+      </div>
+
+      <div *ngIf="isTaskNameEditMode" class="tasks-creator__name-editor">
+
+        <app-input formControlName="taskNameControl"
+                   [noPaddings]="true"
+                   [placeholder]="('TASK-CREATION.TASK_NAME_PLACEHOLDER' | translate)"></app-input>
+
+        <app-icon-button
+          class="tasks-creator__header-button"
+          (click)="disableTaskNameEditMode()"
+          startIcon="done"
+          iconColor="white"
+          iconSize="small">
+        </app-icon-button>
+        <span class="tasks-creator__header-border"></span>
+        <app-icon-button
+          class="tasks-creator__header-button"
+          (click)="clearTaskName()"
+          startIcon="close"
+          iconColor="white"
+          iconSize="small">
+        </app-icon-button>
+
+      </div>
+
+      <app-icon-button
+        (click)="deleteTask()"
+        startIcon="delete"
+        iconColor="white"
+        iconSize="small">
+      </app-icon-button>
+    </div>
+    <div class="tasks-creator__body">
+
+      <div class="tasks-creator__basic-settings">
+
+        <app-select
+          [label]="('TASK-CREATION.TASK_TYPE_LABEL' | translate)"
+          (openedChangeEmitter)="onOpenTaskTypesSelector($event)"
+          [value]="taskTypesEnum.single"
+          (selectionChanged)="rebuildFormControls($event.value)"
+          formControlName="selectedTaskTypeControl"
+          [options]="taskTypeOptions"
+        ></app-select>
+
+        <div class="tasks-creator__mentor-score-container">
+          <app-slide-toggle
+            class="tasks-creator__mentor-check"
+            [labelPosition]="'before'"
```

Comment:
avoid cases like thin across application [...]="'...'" can be transformed to ...="..."